### PR TITLE
[DIGITALRIV-33] Fix updateOrderFormConfiguration response

### PR DESCRIPTION
**Pull request by aurorahey**
What problem is this solving?

The mutation _updateOrderFormConfiguration_ had null responses. This PR fixed the mutation and response with the updated url and decimal 

How to test it?

[graphql](https://aurora--productusqa.myvtex.com/_v/private/vtexus.connector-digital-river@2.3.1/graphiql/v1?query=mutation%20(%24orderFormConfiguration%3A%20OrderFormConfigurationInput!)%20%7B%0AupdateOrderFormConfiguration(orderFormConfiguration%3A%20%24orderFormConfiguration)%20%7B%0A%20%20taxConfigurationUrl%0A%20%20decimalDigitsPrecision%0A%7D%0A%7D&variables=%7B%20%22orderFormConfiguration%22%3A%20%7B%0A%22paymentConfiguration%22%3A%20%7B%0A%22requiresAuthenticationForPreAuthorizedPaymentOption%22%3A%20false%2C%0A%22allowInstallmentsMerge%22%3A%20null%2C%0A%22blockPaymentSession%22%3A%20null%0A%7D%2C%0A%22taxConfiguration%22%3A%20%7B%0A%22url%22%3A%20%22https%3A%2F%2Fsumandigitalriver--productusqa.myvtex.com%2Fdigital-river%2Fcheckout%2Forder-tax%22%2C%0A%22allowExecutionAfterErrors%22%3A%20false%2C%0A%22integratedAuthentication%22%3A%20false%2C%0A%22appId%22%3A%20%22vtexus.connector-digital-river%22%0A%7D%2C%0A%22minimumQuantityAccumulatedForItems%22%3A%201%2C%0A%22decimalDigitsPrecision%22%3A%202%2C%0A%22allowManualPrice%22%3A%20true%2C%0A%22allowMultipleDeliveries%22%3A%20true%0A%7D%0A%7D)

### CODE REVIEW

File: graphql/schema.graphql [hunk 1]
```diff
@@ -7,5 +7,5 @@ type Query {
 type Mutation {
   updateOrderFormConfiguration(
     orderFormConfiguration: OrderFormConfigurationInput!
-  ): OrderFormConfiguration @checkAdminAccess @cacheControl(scope: PRIVATE)
+  ): OrderFormConfigurationResponse @checkAdminAccess @cacheControl(scope: PRIVATE)
```
Comment (AnnaChiu95) [inline graphql/schema.graphql:10]: `updateOrderFormConfiguration` doesn't need to return anything other than `taxConfigurationUrl` and `decimalDigitsPrecision`? Is this mutation only used to update these two fields? If so, maybe the input `OrderFormConfigurationInput` can be restricted to just these two options too.

Comment (aurorahey) [reply graphql/schema.graphql:10]: This POST request originally returns null for successful request, so we basically not returning anything from the curl response. 
All the return value are from the parameter body directly, so we will just return the essentials taxConfigurationUrl and decimalDigitsPrecision from the parameter body. 
We could return all the fields as in OrderFormConfiguration, but that would be nothing relate to the response and directly return whatever in the parameter body.

Comment (AnnaChiu95) [reply graphql/schema.graphql:10]: Ok, so the mutation will still update other fields like `paymentConfiguration` in the orderForm but we only care to see what `taxConfigurationUrl` and `decimalDigitsPrecision` was used in the update?

Comment (aurorahey) [reply graphql/schema.graphql:10]: Correct, and error will be return as response if any of the value not update successfully from the parameter body.

**Review by AnnaChiu95 — APPROVED** [review]
LGTM

**Pull request merged**